Formula/quarantinecli.rb

```diff
@@ -0,0 +1,21 @@
+class Quarantinecli < Formula
+  desc "CLI Utility to quarantine and manage quarantined files on macOS"
+  homepage "https://github.com/Serena-io/quarantineCLI"
+  url "https://github.com/Serena-io/quarantineCLI/archive/refs/tags/1.0.0.tar.gz"
+  version "1.0.0"
```

Comment:
```suggestion
```
This can be read from the `url`, no?

Reply:
Ooh yeah, audit just gave me that, thanks